util/Radiance_Monitor/nwprod/gdas_radmon/fix/gdas_radmon_scaninfo.txt

```diff
@@ -66,5 +67,8 @@
  ahi_himawari9            0.00      1.00        90
  atms_n21               -52.393     1.10        96
  cris-fsr_n21           -48.130     3.333       30
+ viirs-m_npp            -56.280     1.265       90
+ viirs-m_j1             -56.280     1.265       90
+ viirs-m_j2             -56.280     1.265       90
```

Comment:
@RussTreadon-NOAA Thanks for catching this.  I failed to notice that we missed the NSST entries for radmon